langserve/playground/src/App.tsx

```diff
@@ -379,12 +388,9 @@ function App() {
               </ShareDialog>
               <button
                 type="button"
-                className="px-4 py-3 gap-3 font-medium border border-transparent rounded-full flex items-center justify-center bg-blue-500 hover:bg-blue-600 active:bg-blue-700 disabled:opacity-50 transition-colors"
+                className={twMerge("px-4 py-3 gap-3 font-medium border border-transparent rounded-full flex items-center justify-center bg-blue-500 disabled:opacity-50 transition-colors", !isSendDisabled ? "hover:bg-blue-600 active:bg-blue-700" : "")}
```

Comment:
_nit_: there should be a `cn` helper which combines `tailwind-merge` and `clsx`

Reply:
Pushed up